Packs/BmcITSM/Integrations/BmcITSM/README.md

```diff
@@ -438,6 +445,36 @@ Creates a new service request ticket. A service request ticket is the request re
 >|---|---|---|
 >| 2022-07-27T08:44:43 | TAS000000000413 | TAS000000000413 |
 ### bmc-itsm-service-request-update
+
+***
+Updates the details of a service request ticket for a given request ID. User and company related arguments can be retrieved by executing the bmc-itsm-user-list and bmc-itsm-company-list commands.
+
+#### Base Command
+
+`bmc-itsm-service-request-update`
+
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| ticket_request_id | The unique identifier of the service request ticket to update. | Required | 
+| customer_first_name | The customer first name. By default it is determined by the logged in user. | Optional | 
+| customer_last_name | The customer last name. By default it is determined by the logged in user. | Optional | 
+| status | The service request ticket status. Possible values are: Draft, In Cart, In Review, Submitted, Pending, Waiting Approval, Planning, In Progress, Completed, Rejected, Cancelled, Closed. | Optional | 
+| urgency | The ticket request urgency. Required when the ticket creation is without a template. Possible values are: 1-Critical, 2-High, 3-Medium, 4-Low. | Optional | 
+| impact | Incident Request impact. Required when the ticket creation is without a template. Possible values are: 1-Extensive/Widespread, 2-Significant/Large, 3-Moderate/Limited, 4-Minor/Localized. | Optional | 
+| status_reason | The reason for updating the status. Required only if status argument is provided. Possible values are: Review, Need More Information, Approval, System Error, With Issues, Automatically Closed, Successful, By User, By Provider, System, Cancelled, Reopen By User. | Optional | 
+| location_company | The company associated with the service request process. | Optional | 
+| region | The region associated with the company location. | Optional | 
+| site_group | The site group associated with the region. | Optional | 
+| site | The site associated with the site group. | Optional | 
+| assignee | The full name of the employee the ticket will be assigned to. It can be retrieved by using the bmc-itsm-user-list command. | Optional | 
+| additional_fields | The fields which are not present in the current argument list can be added here in the format "fieldname1=value;fieldname2=value". Possible fields: Assigned Group, Assignee, or any other custom field. | Optional | 
+
+#### Context Output
+
+There is no context output for this command.
+### bmc-itsm-service-request-update
```

Comment:
This looks like a duplicate of a pre-existing command. There are more command docs that have this duplication. Let's remove them.